src/algebra/CommRing/colimits.lean

```diff
@@ -293,14 +293,14 @@ begin
   apply relation.map,
 end
 
-@[simp] lemma cocone_naturality_components (j j' : J) (f : j ⟶ j') (x : F.obj j) :
-  (F.map f ≫ (cocone_morphism F j')) x = (cocone_morphism F j) x :=
-by rw cocone_naturality
+@[simp] lemma cocone_naturality_components (j j' : J) (f : j ⟶ j') (x : F.obj j):
+  (cocone_morphism F j') (F.map f x) = (cocone_morphism F j) x :=
+by { rw ←cocone_naturality F f, refl, }
```

Comment:
```suggestion
by { rw ←cocone_naturality F f, refl }
```

Reply:
Sure, I guess. I tend to put in the trailing commas while writing a tactic script so I can inspect the goal as I go, and I'm not sure there's huge value in removing them when you're done.